doc/api/crypto.md

```diff
@@ -1241,18 +1241,14 @@ input.on('readable', () => {
 added: v0.1.92
 -->
 
-Creates and returns a `Sign` object that uses the given `algorithm`. On
-recent OpenSSL releases, `openssl list-public-key-algorithms` will
-display the available signing algorithms. One example is `'RSA-SHA256'`.
+Creates and returns a `Sign` object that uses the given `algorithm`. For available signing algorithms, please reference `crypto.getHashes()`.
```

Comment:
@sstern6 The linking is done via markdown. @jasnell is basically saying instead of using this markdown:

```
`crypto.getHashes()`
```

...use this markdown instead:

```
[`crypto.getHashes()`][]
```

Then make sure that the link is specified in the list of links at the bottom of the markdown document.